app/controllers/volume_mapping_controller.rb

```diff
@@ -3,12 +3,22 @@ class VolumeMappingController < ApplicationController
   include Mixins::GenericShowMixin
   include Mixins::GenericSessionMixin
   include Mixins::BreadcrumbsMixin
+  include Mixins::GenericFormMixin
+  include Mixins::GenericButtonMixin
```

Comment:
looks like you don't need to include button mixin

Reply:
Yep, works without it.